realm/src/main/java/io/realm/BaseRealm.java

```diff
@@ -415,6 +415,19 @@ public void close() {
     }
 
     /**
+     * Check if the {@link io.realm.Realm} instance has already been closed.
```

Comment:
I assume this also works for `DynamicRealm` ? So better to say {@link BaseRealm} here?